src/types/onyx/Card.ts

```diff
@@ -2,43 +2,103 @@ import type {ValueOf} from 'type-fest';
 import type CONST from '@src/CONST';
 import type * as OnyxCommon from './OnyxCommon';
 
+/** Model of expensify card */
 type Card = {
+    /** Card ID number */
     cardID: number;
+
+    /** Current card state */
     state: ValueOf<typeof CONST.EXPENSIFY_CARD.STATE>;
+
+    /** Bank name */
     bank: string;
+
+    /** Available amount to spend */
     availableSpend: number;
+
+    /** Domain name */
     domainName: string;
+
+    /** Last four Primary Account Number digits */
     lastFourPAN?: string;
+
+    /** Determines if the current card was flagged as fraud */
     fraud: ValueOf<typeof CONST.EXPENSIFY_CARD.FRAUD_TYPES>;
```

Comment:
I think the comment is giving the impression the property is some kind of boolean flag, which it isn't.